.gitmodules

```diff
@@ -11,3 +11,6 @@
 [submodule "third_party/bootgen"]
 	path = third_party/bootgen
 	url = https://github.com/Xilinx/bootgen.git
+[submodule "third_party/aie_api"]
+	path = third_party/aie_api
+	url = https://github.com/jgmelber/aie_api
```

Comment:
This is the best URL?

Reply:
For now, I have a fork that comments out an include to adf.h. A follow on will remove any defines to __AIENGINE__, that will gate the include. We use the deprecated convention that flag defines something about the version of the AIE version and redundantly define the correct __AIE_ARCH__.